backend/controller/ingress/handler.go

```diff
@@ -65,25 +82,33 @@ func Handle(
 
 	resp, err := call(r.Context(), creq, optional.Some(requestKey), optional.None[model.RequestKey](), r.RemoteAddr)
 	if err != nil {
-		logger.Errorf(err, "failed to call verb %s", route.Verb)
+		logger.Errorf(err, "failed to call verb")
+		var errMsg string
 		if connectErr := new(connect.Error); errors.As(err, &connectErr) {
 			httpCode := connectCodeToHTTP(connectErr.Code())
 			http.Error(w, http.StatusText(httpCode), httpCode)
-			observability.Ingress.Request(r.Context(), r.Method, r.URL.Path, optional.Some(verbRef), startTime, optional.Some("failed to call verb: connect error"))
+			errMsg = fmt.Sprintf("%s: connect error: %s", err.Error(), connectErr.Message())
 		} else {
 			http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
-			observability.Ingress.Request(r.Context(), r.Method, r.URL.Path, optional.Some(verbRef), startTime, optional.Some("failed to call verb: internal server error"))
+			errMsg = fmt.Sprintf("%s: internal server error", err.Error())
 		}
+		observability.Ingress.Request(r.Context(), r.Method, r.URL.Path, optional.Some(verbRef), startTime, optional.Some(errMsg))
```

Comment:
Will do! I was under the impression that attributes changing would not increase billing, but it sounds like that's not the case so thanks for catching this! I was mostly trying to avoid all the duplication here, but sounds that duplication is useful in this case.

I'll try to clean this up with some helper functions.

Reply:
Ahhh that makes sense. Thanks thanks!